src/components/checkbox/examples/checkbox.tsx

```diff
@@ -32,41 +25,49 @@ export class CheckboxExample {
                         id="fab"
                         checked={this.value}
                         required={this.required}
-                        onChange={this.changeHandler}
+                        onChange={this.handleChange}
                     />
+                </div>
+                <p>
                     <limel-flex-container justify="end">
-                        <limel-button
-                            onClick={this.onClickToggleEnabled}
-                            label={this.disabled ? 'Enable' : 'Disable'}
+                        <limel-checkbox
```

Comment:
Haha! This is pretty meta 😄

Reply:
😝 for consistency. At some point of time, I want to make all these "settings" sections in examples look the same, and be visually separated from the component itself.

Reply:
The "Composite" examples are all quite similar and quite easy to implement
https://github.com/Lundalogik/lime-elements/blob/main/src/components/button-group/examples/button-group-composite.tsx

Some of them can get quite complex however (not in the implementation but the settings UI :smile: )
https://github.com/Lundalogik/lime-elements/blob/main/src/components/menu/examples/menu-composite.tsx
https://github.com/Lundalogik/lime-elements/blob/main/src/components/chip-set/examples/chip-set-composite.tsx

Reply:
I'm thinking maybe we can build a component that uses the schema, but can build and display a nicer form for a number of common and "known" properties, and then just show any "left over" properties underneath, just like we do in the composite examples…? 🤔

Reply:
Yeah sure, that would be possible :slightly_smiling_face: We could just modify the schema a bit before we use it in the form, hiding those settings in a collapsible section

Reply:
sounds great

Reply:
Feels like a potential Hack-Tuesday… 🙂

Reply:
👩‍💻🧑‍💻👨‍💻